tests/test_webvtt_conversion.py

```diff
@@ -34,8 +36,10 @@ def test_positioning_is_kept(self):
         caption_set = WebVTTReader().read(
             SAMPLE_WEBVTT_FROM_DFXP_WITH_POSITIONING)
         results = WebVTTWriter().write(caption_set)
+	# TODO: Change this to SAMPLE_WEBVTT_FROM_DFXP_WITH_POSITIONING once
```

Comment:
Hey 	pkerling, thanks for contributing! Can you add a test for your feature though? You can leave this one unchanged but add a test that uses the sample WITH styling. Ideally you should add new test instead of changing existing ones (unless they're broken by the new feature, of course), so I'd suggest adding a "SAMPLE_WEBVTT_FROM_DFXP_WITH_POSITIONING_AND_STYLE" instead of changing the existing sample. Reading tests is the quickest way for us to give a first judgement of whether a feature is working or not.